airbyte-workers/src/main/java/io/airbyte/workers/protocols/airbyte/AirbyteMessageTracker.java

```diff
@@ -143,6 +159,25 @@ private void handleDestinationEmittedState(final AirbyteStateMessage stateMessag
     }
   }
 
+  /**
+   * When a connector emits a trace message, check the type and call the correct function. If it is an
+   * error trace message, add it to the list of errorTraceMessages for the connector type
+   */
+  private void handleEmittedTrace(final AirbyteTraceMessage traceMessage, final Enum connectorType) {
```

Comment:
nit: use the explicit `ConnectorType` type here rather than the generic `Enum` type - this change will enforce that this method should only ever take in a `ConnectorType` enum value

Same goes for `handleEmittedErrorTrace` below